api/src/pcapi/routes/backoffice_v3/templates/accounts/get/details/bookings.html

```diff
@@ -68,6 +68,10 @@
                 <span class="fw-bold">Crédit utilisé par le jeune :</span>
                 {{ booking.deposit.type | format_deposit_type | safe }}
               </p>
+              <p class="mb-1">
+                <span class="fw-bold">Email à la réservation :</span>
```

Comment:
J'ai l'impression qu'on a tendance à passer à E-mail, maintenant
```suggestion
                <span class="fw-bold">E-mail à la réservation :</span>
```